### PR TITLE
Navigation component

### PR DESCRIPTION
**Pull request by amyria3**

### CODE REVIEW

File: src/components/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+function Navigation() {
```
Comment (ReallyBoard123) [inline src/components/Navigation.tsx:1]: can you please refactor these using HeroIcons or some similar icon library?

**Comment by amyria3** [comment]
hey I don't understand why the last changes are not visible in [29dc198](https://github.com/devhausleipzig/camp-8-midterm-group-2/pull/25/commits/29dc19871584e70f5be68ead9ea271f137c52d1a)
as in my local branch its up to date and I pushed several times just to be sure and whenever I try git says its all up-to date :)

File: src/layouts/NavBarLayout.tsx [hunk 2]
```diff
@@ -1,15 +1,59 @@
...
-  return (
-    <div>
+type IsActive = {
+  home: boolean;
+  movies: boolean;
+  wallet: boolean;
+  account: boolean;
+};
+
+let isActive: IsActive = {
```
Comment (ReallyBoard123) [inline src/layouts/NavBarLayout.tsx:13]: this needs to be used as state so that it is reactive

**Review by NoHara42 — APPROVED** [review]
LGTM, nice fix for isActive

**Comment by NoHara42** [comment]
@amyria3 please reverse merge (by merging main into this branch) and resolve the merge conflict before I merge to main

**Pull request merged**